src/styles.scss

```diff
@@ -4,6 +4,10 @@
 @import './scss/variables.scss';
 @import './scss/utils.scss';
 
+:root {
+  scroll-behavior: smooth;
```

Comment:
Isn't it too early to use this property?

Reply:
Because is not fully [implemented](https://caniuse.com/#search=scroll-behavior) ?

Reply:
Quoting the css tricks article [_Smooth Scrolling and Accessibility_](https://css-tricks.com/smooth-scrolling-accessibility/#article-header-id-0)

>  when you follow a link, the keyboard focus should follow it, too and be able to access the next element after the target. 

Here is an example

### [demo](https://output.jsbin.com/yilokec)
![example](https://css-tricks.com/wp-content/uploads/2017/04/example-2-resized.gif)

#### To implement a smooth scrolling with that recommendation the best way that I could find is with that css property. 

----------------

## Why?
The current implementation of the smooth scrolling do not update the keyboard focus.
```js
elementRef.scrollIntoView({	
  behavior: 'smooth',	
  block: 'start'	
});
```

There is a lack of callback to know when the scroll finish to update the focus to the new element and if you do it right away like this
```js
elementRef.scrollIntoView({	
  behavior: 'smooth',	
  block: 'start'	
});
elementRef.focus();
```
When you `elementRef.focus()`, the target hijacks the smooth scrolling and it jumps right at the target, abruptly.

----------------

The possible solutions that I found were:
- Use Jquery animate
- Implement a `setTimeout` with the same duration as the scrolling
- Use the css property `:root { scroll-behavior: smooth; }`

I don't want to add a library like jquery just for that, making the web page load slow.
Agreeing with a [comment](https://css-tricks.com/smooth-scrolling-accessibility/#comment-1608095) in the article, use a `setTimeout` doesn't feel particularly elegant.

If you look at [Can I Use](https://caniuse.com/#search=scroll-behavior) the porcentaje of usage is 71.42% considering all users (desktop and mobile), doesn't look too bad like the other options.

![image](https://user-images.githubusercontent.com/7026066/58196476-01de6e00-7c90-11e9-91ba-8fb60879cd01.png)

Reply:
There is something that is not clear o I need to go deeper ?

Reply:
The idea is great. I'm just afraid about browser compatibility for now. 71% isn't that much

Reply:
The 29% left, just is not going to have a smooth scroll, the functionality is going to be the same. Their jump is going to be immediate

Reply:
You can make the test in an old browser (chrome 59) that doesn't support the css property

https://www.browserling.com/browse/win/7/chrome/59/https%3A%2F%2Fdeploy-preview-433--docs-nestjs.netlify.com%2F

Reply:
It rises up to [75.97%](https://caniuse.com/#search=scroll-behavior)